Function mass_density moved from misc.py to densities.py (#2410)
PlasmaPy/PlasmaPy: changelog/2410.breaking.rst

> Removed the ``plasmapy.formulary.misc.mass_density`` function. This

Reviewer: Very good catch 👍

> function is now in `plasmapy.formulary.densities`.

Reviewer 2: ```suggestion
function is now at `plasmapy.formulary.densities.mass_density`.
```

> Removed the ``plasmapy.formulary.misc.mass_density`` function. This
> function is now in `plasmapy.formulary.densities`.

Reviewer 2: ```suggestion
Moved ``mass_density`` and its alias ``rho_`` from `plasmapy.formulary.misc`
to `plasmapy.formulary.densities`.
```
